lldb/source/Expression/IRMemoryMap.cpp

```diff
@@ -640,6 +640,13 @@ void IRMemoryMap::WritePointerToMemory(lldb::addr_t process_address,
                                        lldb::addr_t address, Status &error) {
```

Comment:
Also please do rename one of these parameters to be pointer, it's incredibly unclear what is what here. (in another PR ofc, or push directly, it's trivial)

Reply:
I'll do this after this PR is merged, otherwise I'll have to rebase and force push to this PR, which would lead to losing line attributions to review comments.